core/services/chainlink/application.go

```diff
@@ -89,9 +89,9 @@ func NewApplication(config *orm.Config, onConnectCallbacks ...func(Application))
 	runExecutor := services.NewRunExecutor(store, statsPusher)
 	runQueue := services.NewRunQueue(runExecutor)
 	runManager := services.NewRunManager(runQueue, config, store.ORM, statsPusher, store.TxManager, store.Clock)
-	jobSubscriber := services.NewJobSubscriber(store, runManager)
 	gasUpdater := services.NewGasUpdater(store)
 	logBroadcaster := eth.NewLogBroadcaster(store.TxManager, store.ORM, store.Config.BlockBackfillDepth())
+	jobSubscriber := services.NewJobSubscriber(store, runManager, logBroadcaster)
```

Comment:
Why has it moved?

Reply:
It's only because the jobSubscriber now needs the logBroadcaster.